### PR TITLE
[browser][wasm] Add Function tests.

### PR DESCRIPTION
**Pull request by kjpou1**

### CODE REVIEW

File: src/libraries/System.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/JavaScriptTests.cs [hunk 1]
```diff
@@ -80,10 +80,41 @@ public static void CoreTypes()
...
             Assert.Equal(8, (int)sum.Call(null, 3, 5));
+
+            Assert.Equal(13, (int)sum.Apply(null, new object[] { 6, 7 }));
+        }
+
+        [Fact]
+        public static void FunctionMath()
+        {
+            JSObject math = (JSObject)Runtime.GetGlobalObject("Math");
+            Assert.NotNull (math);
```
Comment (MaximLipnin) [inline src/libraries/System.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/JavaScriptTests.cs:99]: It looks like Mono style 😃

Comment (kjpou1) [reply src/libraries/System.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/JavaScriptTests.cs]: Formatted

**Review by akoeplinger — APPROVED** [review]

File: src/libraries/System.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/JavaScriptTests.cs [hunk 2]
```diff
@@ -56,34 +56,65 @@ public static void CoreTypes()
...
+            Function mathMax = (Function)math.GetObjectProperty("max");
+            Assert.NotNull(mathMax);
+
+            var maxValue = (int)mathMax.Apply(null, new object[] { 5, 6, 2, 3, 7 });
+            Assert.Equal(7, maxValue);
+
+            maxValue = (int)mathMax.Call(null, 5, 6, 2, 3, 7);
+            Assert.Equal(7, maxValue);
+
+            Function mathMin = (Function)((JSObject)Runtime.GetGlobalObject("Math")).GetObjectProperty("min");
```
Comment (MaximLipnin) [inline src/libraries/System.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/JavaScriptTests.cs:110]: We already have Math object at line 98

**Review by MaximLipnin — APPROVED** [review]

File: src/libraries/System.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/JavaScriptTests.cs (hunk 2 above)
Comment (kjpou1) [reply src/libraries/System.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/JavaScriptTests.cs:110]: That was to test for concatenation as well.  We can break that out in other tests and make a specific test instead if that is less confusing.

**Pull request merged**